apis/pipeline/pipeline.yaml

```diff
@@ -4494,6 +4494,45 @@ components:
                     type: integer
                     example: 5
 
+        AWSAuthConfigMap:
```

Comment:
How about one of the following (or similar):

- `AWSAuth`
- `AWSAuthConfig`
- `AWSClusterAuth`
- `AWSClusterAuthConfig`

Reply:
I changed it to `AWSAuthConfig`